examples/walkthrough/k8s201.md

```diff
@@ -48,15 +95,40 @@ spec:
             - containerPort: 80
 ```
 
-### Services
+#### Replication Controller Management
+
+If you have the source checked out or downloaded the pre-compiled release you should have an `examples` directory in the Kubernetes root directory. If not, download the [nginx replication controller example definition](replication-controller.yaml).
+
+Create an nginx replication controller:
+
+```sh
+kubectl create -f examples/walkthrough/replication-controller.yaml
+```
+
+List all replication controllers:
+
+```sh
+kubectl get rc
+```
+
+Delete the replication controller by name:
+
+```sh
+kubectl delete rc nginx-controller
+```
+
+
+## Services
+
 Once you have a replicated set of pods, you need an abstraction that enables connectivity between the layers of your application.  For example, if you have a replication controller managing your backend jobs, you don't want to have to reconfigure your front-ends whenever you re-scale your backends.  Likewise, if the pods in your backends are scheduled (or rescheduled) onto different machines, you can't be required to re-configure your front-ends.  In Kubernetes, the Service object achieves these goals.  A Service basically combines an IP address and a label selector together to form a simple, static rallying point for connecting to a micro-service in your application.
```

Comment:
Better: ... basically like a simple load-balancer in front of that service with a stic IP.